website/docs/d/workspace.html.markdown

```diff
@@ -34,6 +34,8 @@ In addition to all arguments above, the following attributes are exported:
 * `allow_destroy_plan` - Indicates whether destroy plans can be queued on the workspace.
 * `auto_apply` - Indicates whether to automatically apply changes when a Terraform plan is successful.
 * `file_triggers_enabled` - Indicates whether runs are triggered based on the changed files in a VCS push (if `true`) or always triggered on every push (if `false`).
+* `global_remote_state` - (Optional) Whether to allow remote state access to this workspace. It defaults to `true`.
```

Comment:
This seems to imply that `false` means remote state access isn't allowed, but it's just for implicit org-wide sharing.

I'd just use what's in the API docs, tweaked slightly for the provider:

```suggestion
* `global_remote_state` - (Optional) Whether the workspace should allow all workspaces in the organization to access its state data during runs. If false, then only specifically approved workspaces can access its state (determined by the `remote_state_consumer_ids` argument).
```